<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1261</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>1181</width>
      <height>431</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QTableView" name="fna_files_selected_table">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>50</y>
       <width>741</width>
       <height>331</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Database</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>30</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>File Selected</string>
     </property>
    </widget>
    <widget class="QPushButton" name="select_button">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>390</y>
       <width>181</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Select Files</string>
     </property>
    </widget>
    <widget class="QPushButton" name="deselect_button">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>390</y>
       <width>181</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Deselect Files</string>
     </property>
    </widget>
    <widget class="QTableView" name="fna_files_unselected_table">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>351</width>
       <height>331</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="merge_button">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>470</y>
      <width>181</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Merge Selected Files</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1261</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
